src/Dependencies/Composition/project.json

```diff
@@ -0,0 +1,9 @@
+{
+  "supports": {},
+  "dependencies": {
+    "Microsoft.Composition": "1.0.27"
```

Comment:
So that we have the version in one file vs in many. Updating to a new version is easier.